mira/sources/askenet/flux_span.py

```diff
@@ -126,11 +126,17 @@ def reproduce_ode_semantics(flux_span):
                                            [reverse_map_1, reverse_map_2]):
         for key, ic in original_model.initials.items():
             concepts = reverse_map[ic.concept.name]
-            original_value = ic.value
+            original_value = ic.expression
             for concept in concepts:
-                new_initial_conditions[concept] = \
+                try:
```

Comment:
Same here, I don't understand why we need try/except here just to divide an expression. You can apply the division to any sympy.Expr object, and arg[0] of SympyExprStr should always be a sympy.Expr.